module/tabbed.lua

```diff
@@ -173,12 +173,27 @@ tabbed.update_tabbar = function(tabobj)
     end
     -- add tabbar to each tabbed client (clients will be hided anyway)
     for _, c in ipairs(tabobj.clients) do
+
+        local bar_shape = helpers.shape.rrect(0)
+        if bar.position == "top" then
+            bar_shape = helpers.shape.prrect(beautiful.tabbar_corner_radius,
+                                             true, true, false, false)
+        elseif bar.position == "bottom" then
+            bar_shape = helpers.shape.prrect(beautiful.tabbar_corner_radius,
+                                             false, false, true, true)
+        end
+
```

Comment:
The tabbar position could also be on the left or on the right. I think only `boxes` allows for that currently.

Reply:
forgot that people use vertical titlebars and wibars

Reply:
@Nooo37 done